src/node/rpc/frontend.h

```diff
@@ -688,6 +721,10 @@ namespace ccf
       std::shared_ptr<enclave::RpcContext> ctx) override
     {
       auto tx = tables.create_tx();
+      if (!is_open(tx))
+      {
+        throw std::logic_error("Transaction failed");
```

Comment:
Please add a comment here that this state can only be reached with a malicious primary so we should trigger a view change when catching this

Reply:
Done.